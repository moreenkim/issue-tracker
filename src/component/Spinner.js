import React from 'react';
import { Spin } from 'antd';

import './Spinner.css';

const Spinner = () => {
  return (
    <div id="spinner">
      <Spin />
    </div>
  );
};

export default Spinner;
