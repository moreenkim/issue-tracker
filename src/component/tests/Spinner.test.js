// Test Spinner
import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import Spinner from '../Spinner';

describe('components/Spinner', () => {
  it('renders without crashing', () => {
    shallow(<Spinner />);
  });

  it('renders spinner correctly', () => {
    const wrapper = mount(<Spinner />);
    expect(toJson()).toMatchSnapshot();
    wrapper.unmount();
  });
});
