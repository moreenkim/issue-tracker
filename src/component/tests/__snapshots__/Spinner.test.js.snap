// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Spinner renders spinner correctly 1`] = `undefined`;
